bigbluebutton-html5/imports/ui/components/banner-bar/container.jsx

```diff
@@ -0,0 +1,8 @@
+import { Session } from 'meteor/session';
+import { withTracker } from 'meteor/react-meteor-data';
+import BannerComponent from './component';
+
+export default withTracker(() => ({
+  color: Session.get('bannerColor') || undefined,
```

Comment:
Does `Session.get` returns something other than `undefined` for undefined keys? 🤔

Reply:
I did that to trigger the default props, `session.get` return `null` when no there the value.

Reply:
Gotcha but that's not implicit. I would suggest to remove the `defaultProps` and make them required. And then swap the `undefined` for the default values.